homeassistant/components/smartthings/sensor.py

```diff
@@ -360,3 +365,61 @@ def state(self):
             return three_axis[self._index]
         except (TypeError, IndexError):
             return None
+
+
+class SmartThingsPowerConsumptionReportSensor(SmartThingsEntity):
+    """Define a SmartThings Power Consumption Report."""
+
+    def __init__(self, device):
+        """Init the class."""
+        super().__init__(device)
+        self._attribute = Attribute.power_consumption
+        self._name = "Power Consumption Report"
```

Comment:
If we name this "Power Consumption Report" then the sensor will be something like ```sensor.samsung_dryer_power_consumption_report``` and the attributes will be ```sensor.samsung_dryer_power_consumption_report.power_consumption_{ attr }```
I'd favor something less verbose - maybe naming the sensor "Report" instead of "Power Consumption Report"?

Reply:
I suggest we keep as close to the capability and attributes as we can. The library prepends `power_consumption` to all of the inner attributes because it's hanging off the root status - we should use a map to remove that unnecessary prefix.

Reply:
I added a map to the const file:
https://github.com/home-assistant/core/pull/43062/commits/4f7ca579a4d51fd7926ac0b4b8f92f99f243f3b0#diff-fbdce5043f7cc31204f060929107e1a7cb908aa172047999cfc28198aaedab7dL21-R31
Notably, this does not map "energy" since it is the state of the sensor.